test/apps/react-routing/react-router-dom-v5-hash/README.md

```diff
@@ -0,0 +1,21 @@
+# Okta React Router DOM v5 Test App
+
+## Install
+```bash
+$ git clone https://github.com/okta/okta-react.git
+$ yarn
+$ cd ./test/apps/react-routing/react-router-dom-v5
```

Comment:
If this sample is added to yarn workspace, you can use `yarn workspace @okta/test.app.react-router-v5-hash start`. In this way, we will not need to keep maintaining the path whenever it changes in the future.